content/project/board-election-process.adoc

```diff
@@ -1,37 +1,135 @@
+. Have election every year, normally electing 2 people in each year (that means the term is 2 years, without term limit: you can be a candidate and be re-elected indefinitely)
+. There will be a period of at least 3 weeks designated for the board to encourage the community to send in the nominations. The board will meet in private to come up with the list of candidates at the end
 . Any company should not dominate in the board (link:/project/board-election-process/#corporate-involvement[see below])
 .. The number of board members affiliated with one company must be less than 50%
 .. If a board member gets employed by a company and the limitation gets violated, somebody must step down and the board will follow the link:/project/board-election-process/#interim-procedures[#Interim Procedures]
 . Use link:https://civs.cs.cornell.edu/[Condorcet Internet Voting Service] as the algorithm to elect multiple people in one vote.
-. You earn the voting right by having an account on jenkins-ci.org prior to the last election. (to avoid last minute rush since we have few other criteria for voters.) For the first election, we will create the election cut-off based on the date when this proposal is ratified.
 
-## Interim Procedures
-
-. If a board member resigns, the board is allowed to appoint an interim board member to fulfill the remainder of the term, subject to blessing in a link:https://wiki.jenkins.io/display/JENKINS/Governance+Meeting+Agenda[regular governance meeting].
+### Nominations
+
+The opinion of members of the community at large is highly valued, and the board welcomes additional nominations beyond who we might consider.
+If you feel that a particular person is well suited to help guide Jenkins, please submit a name and the reason for your nomination to jenkinsci-board@googlegroups.com.
+
+The election committee will review all submissions and compile a final list of candidates to be voted on by the community, for the board member seats and officer roles up for election.
+The board reserves the right to leave nominations off the final ballot if they feel that a particular viewpoint is already well represented on the board,
+or if there are other reasons preventing a nominee from being effective in the target role.
+In the case of rejection, a justification must be provided to the contributor who submitted the nomination.
+The decision may be escalated to the link:/project/governance-meeting/[Jenkins governance meeting] and overridden by voting there.
+
+After receiving and vetting the nominations,
+the election committee is responsible to reach out to nominees and to confirm whether they will be candidates in the upcoming elections.
+The candidates should also provide their statements, list of affiliations (see _Corporate Disclosure_), and account/social media links which may help voters to get more information about the candidates and plans if they get elected to the role.
```

Comment:
I think that the `<<` and `>>` will convert this into a hyperlink to the relevant section.
```suggestion
The candidates should also provide their statements, list of affiliations (see <<Corporate Disclosure>>), and account/social media links which may help voters to get more information about the candidates and plans if they get elected to the role.
```